dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -52,25 +59,47 @@
 class StaticLinkageChecker {
 
   /**
-   * Given list of the jar files as file names in filesystem, outputs the report of static linkage
-   * check.
+   * Given one Maven coordinate or list of the jar files as file names in filesystem, outputs the
+   * report of static linkage check.
    *
-   * @param arguments jar files available in the file system
+   * @param arguments one Maven artifact, or multiple jar files available in the file system
    * @throws IOException when there is a problem in reading a jar file
    * @throws ClassNotFoundException when there is a problem in reading a class from a jar file
+   * @throws RepositoryException when there is a problem in resolving the Maven coordinate to jar
+   *     files
    */
-  public static void main(String[] arguments) throws IOException, ClassNotFoundException {
+  public static void main(String[] arguments)
+      throws IOException, ClassNotFoundException, RepositoryException {
+    if (arguments.length < 1) {
+      System.out.println("No argument specified.");
+      return;
+    }
+    List<Path> jarFilePaths;
+    if (arguments.length == 1) {
```

Comment:
what if there's a single jar file? What if there are multiple maven coordinates? We probably need a more complete CLI here with arguments and such.

Reply:
Let me implement it with commons-cli.

Reply:
Done